css/style.css

```diff
@@ -65,4 +65,10 @@
 
 form {
     padding: 0.5em;
+}
+
+#edit-submit {
+    position: fixed;
```

Comment:
position fixed?!?! why?

Reply:
The button is more accesible in this way, because there is many fields, and maybe you don't want to modify or see al of them.